src/main/resources/org/jenkinsci/plugins/sonargerrit/SonarToGerritPublisher/config.properties

```diff
@@ -5,6 +5,8 @@ jenkins.plugin.settings.section.filter.name=Filter Settings
 jenkins.plugin.settings.filter.severity=Report issues having severity level higher or equal to:
 jenkins.plugin.settings.filter.new=Report new issues only?
 jenkins.plugin.settings.filter.new.description=Reports new as well as existing issues when unchecked
+jenkins.plugin.settings.filter.failonly=Report only on failure?
+jenkins.plugin.settings.filter.failonly.description=Reports only failures when checked
```

Comment:
Usually in Jenkins stuff "failure" (personally for me) means "build **_failure_**" not, like I think in this case Sonar **_issue_**.